JS-Algorithms/challenges.js

```diff
@@ -12,8 +13,17 @@ Example:
 Invoking "readableTime(3690)" should return "01:01:30" (HH:MM:SS)
 ***** */
 
+const dateFormat = (value) => {
+  return value < 10 ? `0${value}` : value;
+}
+
 const readableTime = (seconds) => {
-  // YOUR CODE HERE...
+
+  let hours = parseInt(seconds / 3600);
+  let minutes = parseInt(seconds / 60) % 60;
+  seconds = parseInt(seconds % 60);
```

Comment:
we should not reasign values from parameters, instead make a copy of the parameter in another variable and then operate on this

Reply:
fixed